CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- Add button to allow users to create form a MAT projects
+- Add button to allow users to create form a MAT conversion projects
```

Comment:
This should be transfers.

Reply:
I edited this line to make it clear it's conversions - it's from an older commit. I added a new one for transfers!